translations/es/LC_MESSAGES/messages.po

```diff
@@ -1094,7 +1094,7 @@ msgstr ""
 
 #: /content/includes/menu.yaml
 msgid "Videos"
-msgstr ""
+msgstr "Videos"
```

Comment:
Translation to same word.
A long time ago I translated that words, and you rejected that changes :-1: